src/models/index.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+var fs = require('fs')
+var path = require('path')
+var Sequelize = require('sequelize')
+var basename = path.basename(module.filename)
+var env = process.env.NODE_ENV || 'development'
+var config = require(__dirname + '/../config/config.js')[env]
+var db = {}
+
+if (config.use_env_variable) {
+  var sequelize = new Sequelize(process.env[config.use_env_variable])
+} else {
+  var sequelize = new Sequelize(
+    config.database,
+    config.username,
+    config.password,
```

Comment:
This bit is not being used

Reply:
It could be used in a different environment e.g. testing